cpp/box/Box.h

```diff
@@ -381,6 +382,53 @@ class Box
         });
     }
 
+    //! Calculate distance between two points using boundary conditions
+    /*! \param p_i Particle position
+        \param p_j Particle position to calculate distances from
+    */
+    float computeDistance(const vec3<float>& p_i, const vec3<float>& p_j) const
```

Comment:
I would prefer `r_i` over `p_i` for naming consistency with the rest of freud's code base.